data/experiment-recipe-samples/rollout-mobile.json

```diff
@@ -0,0 +1,38 @@
+{
+  "schemaVersion": "0.1.0",
+  "id": "rollout-mobile-example",
+  "slug": "rollout-mobile-example",
+  "application": "reference_browser",
+  "appId": "org.mozilla.reference.browser",
+  "appName": "reference_browser",
+  "featureIds": ["bookmark-icon"],
+  "channel": "nightly",
+  "userFacingName": "Mobile Rollout Example",
+  "userFacingDescription": "An A/A Test to validate rollout in the Rust SDK",
+  "isRollout": true,
+  "isEnrollmentPaused": false,
+  "bucketConfig": {
+    "randomizationUnit": "nimbus_id",
+    "namespace": "rollout-mobile-example-rollout-1",
+    "start": 0,
+    "count": 5000,
+    "total": 10000
+  },
+  "targeting": null,
+  "startDate": null,
+  "endDate": null,
+  "proposedEnrollment": 7,
+  "referenceBranch": "control",
+  "probeSets": [],
+  "branches": [
+    {
+      "slug": "control",
+      "ratio": 1,
+      "feature": {
+        "featureId": "bookmarkId",
```

Comment:
Similar to my comment above, I have not seen this feature in Stage.

Reply:
Do you have sample data that could be used here instead? This is just a copy of [another data file in this repo](https://github.com/mozilla/nimbus-shared/blob/main/data/experiment-recipe-samples/mobile-a-a.json) with `isRollout: true` and a few other minor changes.